File: 01- Array - String/C#/6- Reverse Words in a String.cs
public class Solution {
    public string ReverseWords(string s) {
        string ans = "" ; 
        for (int i = 0 ; i < s.Length ; i++){
            if (i + 1 < s.Length && s[i] == ' ' && s[i+1] != ' ')
                ans += ' ' ; 
            else if (s[i] != ' ')
                ans += s[i] ; 
        }

        string[] str = ans.Split(" ");
        Array.Reverse(str) ; 
        string sss = string.Join(" " , str);

        return sss.Trim(); 
    }
}

File: 01- Array - String/6- Reverse Words in a String.cpp
class Solution {
public:
    string reverseWords(string s) {
        stringstream ss(s) ;
        string str ;
        vector<string> strs ;

        while (ss >> str)
            strs.push_back(str);

        reverse(strs.begin() , strs.end());

        string ans ;
        for (int i = 0 ; i < strs.size() ; i++) {
            if (i > 0)
                ans += ' ' ;
            ans += strs[i];
        }

        return ans ; 
    }
};